CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ HumHub Changelog
 - Enh #7244: Changed .env var depth separator from `.` to `__`
 - Fix #7248: Upgrade jQuery Highlight plugin
 - Fix #7254: Fix Login view HTML element ID from `user-auth-login-modal` to `user-auth-login`
+- Enh #7253: CSV export improvements
```

Comment:
@gevorgmansuryan Does this only apply to CSV? Do we need some other improvements for XLSX?

Reply:
@luke- While XLSX is more advanced and has protection, we can apply the same sanitizing for XLSX too, there will be no harm. Apply?

Reply:
@luke- if you think it is needed then the following could be used as an example;

```php
/**
 * Sanitize value to prevent injection in CSV and XLSX.
 */
private function sanitizeValue(?string $value): ?string
{
    if (empty($value) || !in_array($this->resultConfig['writerType'], ['csv', 'xlsx'])) {
        return $value;
    }

    // Check for risky starting characters or formula-like values and prepend single quote
    if (strpbrk($value[0], '=+-@,;' . "\t" . "\r") !== false || preg_match('/^\d+[+\-*\/].+/', $value)) {
        $value = "'" . $value;
    }

    // Sanitize escaping quotes, wrapping in double quotes if needed
    if (strpbrk($value, "\"\n,") !== false) {
        $value = '"' . str_replace('"', '""', $value) . '"';
    }

    return $value;
}
```

Reply:
@luke- @gevorgmansuryan I'd like to mention that adding XLSX in an array will increase the execution time but shouldn't be noticeable by the end user.